### PR TITLE
{tools}[GCCcore/12.2.0] fsm-lite v1.0, sdsl-lite v2.0.3

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/f/fsm-lite/fsm-lite-1.0-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+]
+
+prebuildopts = "sed -i '1s/.*/SDSL_INSTALL_PREFIX=${EBROOTSDSLMINLITE}/' Makefile && make depend &&"
+
+files_to_copy = [(['fsm-lite'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/fsm-lite'],
+    'dirs': [],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/f/fsm-lite/fsm-lite-1.0-GCCcore-12.2.0.eb:32]: @verdurin is it possible to add a `sanity_check_commands` item here?

Comment (verdurin) [reply easybuild/easyconfigs/f/fsm-lite/fsm-lite-1.0-GCCcore-12.2.0.eb:32]: @smoors done

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-el8.cloud.in.bmrc.ox.ac.uk - Linux Rocky Linux 8.10, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/2f15829f654a416bfd3fecb2867f6f57 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @verdurin!

**Pull request merged**